core/storage/user/gae_models.py

```diff
@@ -159,7 +159,9 @@ def export_data(user_id):
         Returns:
             dict. Dictionary of the data from UserSettingsModel.
         """
-        user = UserSettingsModel.get(user_id)
+        user = UserSettingsModel.get(user_id, strict=False)
```

Comment:
I actually think that, for the case where UserSettingsModel is None, we should raise an error. It doesn't make sense to export a user who doesn't even exist.

The {} case should be reserved for cases where creating the model for the user is legitimately an optional action.

Reply:
Makes sense! Done.